docs/installation.rst

```diff
@@ -80,7 +80,7 @@ You can follow the steps below to install FlashInfer from source code:
            .. code-block:: bash
 
                cd flashinfer
-               export FLASHINFER_CUDA_ARCH_LIST="7.5 8.0 8.9 9.0a 10.0a"
+               export FLASHINFER_CUDA_ARCH_LIST="7.5 8.0 8.9 9.0a 10.0a 12.0"
```

Comment:
maybe explicitly specifying "12.0a"?

Reply:
We will append an "a" anyways: https://github.com/flashinfer-ai/flashinfer/blob/08b8da35ffc26454415a9b833a69e114a645ede3/flashinfer/compilation_context.py#L40-L41, but maybe this is some behavior we should change.

Reply:
The behavior of compilation context is fixed in: https://github.com/flashinfer-ai/flashinfer/pull/1798

Reply:
> Spark is 12.1, so not sure 12.0a is a good idea...

Then why not `FLASHINFER_CUDA_ARCH_LIST="7.5 8.0 8.9 9.0a 10.0a 12.0a 12.1a`? Or `12.0f`
12.0 without a cannot use features such as TMA.